src/content/en/2021/structured-data.md

```diff
@@ -17,4 +19,568 @@ featured_stat_3: TODO
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3kZ1Ys-9tId-WBa_8muMzrTAu1Ad5TXYgkopXMmBVc1xmd2N_4PZIpEZEOqRL7baymle0kHzaC6KY/pubchart?oid=1501633178&format=interactive",
+  sheets_gid="685929099",
+  width=600,
+  height=638,
+  sql_file="jsonld_same_ases.sql"
+  )
+}}
+
+The `sameAs` property accounts for 1.60% of all JSON-LD markup and is present on 13.03% of pages.
+
+We can see that the most common values of the `sameAs` property (normalizing from _URLs_ to _hostnames_) are social media platforms (e.g., facebook.com, instagram.com), and official sources (e.g., wikipedia.org, yelp.com)—with the sum of the former accounting for ~75% of usage.
+
+It's clear that this property is primarily used to identify the social media accounts of websites and businesses; likely motivated by Google's historical reliance on this data as an input for managing _knowledge panels_ in their search results. Given that this requirement was <a hreflang="en" href="https://twitter.com/googlesearchc/status/1143558928439005184">deprecated in 2019</a>, we might expect this data set to gradually alter in coming years.
+
+## Conclusion
+
+Structured data is used broadly, and diversely, across the web. Whilst some of this is undoubtedly stale (legacy sites/pages, using outmoded formats), there is also strong adoption of new and emerging standards.
+
+Anecdotally, much of the adoption we see of modern standards like schema.org (particularly via JSON-LD) appears to be motivated by organizations and individuals who wish to take advantage of search engines' support (and rewards) for providing data about their pages and content. But outside of this, there's a rich landscape of people who use structured data to enrich their pages for other reasons. They describe their websites and content so that they can integrate with other systems, so that they can better understand content, or in order to facilitate _others_ to tell _their own_ stories and build their own products.
+
+A web made of deeply connected, structured data which powers a more integrated world has long been a science-fiction dream. But perhaps, not for much longer. As these standards continue to evolve, and their adoption continues to grow, we pave a road towards an exciting future.
+
+### Future years
+
+As this is the first time this chapter has been included in the Web Almanac, we don't have any data we can use to explore change over time.
+
+In future years, we'll be able to assess how the distribution, types, and formats of structured data have changed across the web. This may provide valuable insight into growing/declining technologies, strategies, and opportunities.
```

Comment:
Maybe rephrase the first sentence to be more positive?

Something along the lines of "In future years we hope to be able to continue this analysis the analysis started here and map the evolution of structured data usage over time." which sounds more like this is an opportunity, than an excuse or something we've missed or an unable to cover in this chapter?

Reply:
Done